scripts/build-installer-win64.bat

```diff
@@ -69,51 +62,6 @@ call npm install
     exit /b 1)
 :after_node
 
-@if not [%SKIP_CARDANO_FETCH%]==[] (@echo "WARNING: SKIP_CARDANO_FETCH set, not re-fetching Cardano"
-    goto :after_cardano_fetch)
-@echo ##############################################################################
-@echo ###
-@echo ### Obtaining Cardano from branch %CARDANO_BRANCH%
-@echo ###
-@echo ##############################################################################
-rmdir /s/q node_modules\daedalus-client-api 2>nul
-mkdir      node_modules\daedalus-client-api
-
-pushd node_modules\daedalus-client-api
-    del /f CardanoSL.zip 2>nul
-    ..\..\curl --location %CARDANO_URL% -o CardanoSL.zip
-    @if %errorlevel% neq 0 (@echo FAILED: couldn't obtain the cardano-sl package
-	popd & exit /b 1)
-    @for /F "usebackq" %%A in ('CardanoSL.zip') do set size=%%~zA
-    if %size% lss %MIN_CARDANO_BYTES% (@echo FAILED: CardanoSL.zip is too small: threshold=%MIN_CARDANO_BYTES%, actual=%size% bytes
-        popd & exit /b 1)
-popd
-:after_cardano_fetch
-
-pushd node_modules\daedalus-client-api
-    7z x CardanoSL.zip -y
-    @if %errorlevel% neq 0 (@echo FAILED: 7z x CardanoSL.zip -y
-	popd & exit /b 1)
-popd
-
-@echo ##############################################################################
-@echo ###
-@echo ### cardano-sl build-id:
-@type node_modules\daedalus-client-api\build-id
-@echo ### cardano-sl commit-id:
-@type node_modules\daedalus-client-api\commit-id
-@echo ### cardano-sl ci-url:
-@type node_modules\daedalus-client-api\ci-url
-@echo ###
-@echo ##############################################################################
-
-move   node_modules\daedalus-client-api\log-config-prod.yaml installers\log-config-prod.yaml
-move   node_modules\daedalus-client-api\cardano-node.exe     installers\
-move   node_modules\daedalus-client-api\cardano-launcher.exe installers\
-move   node_modules\daedalus-client-api\configuration.yaml installers\
-move   node_modules\daedalus-client-api\*genesis*.json installers\
-del /f node_modules\daedalus-client-api\*.exe
-
```

Comment:
This is pure happiness of a diff chunk! : -)

Reply:
There is more which can and should be removed from the `.bat` file.